Step-by-step-Tutorials/My-First-Orleans-Application.md

```diff
@@ -53,7 +53,9 @@ static void Main(string[] args)
         AppDomainInitializerArguments = args,
     });
 
-    Orleans.GrainClient.Initialize("DevTestClientConfiguration.xml");
+    // Orleans comes with a rich XML configuration but we're just going to setup a basic config
+    var config = new Orleans.Runtime.Configuration.ClientConfiguration();
+    GrainClient.Initialize(config);
```

Comment:
agreed.
BTW, notice that even though documentation wasn't updated, all [samples](https://github.com/dotnet/orleans/tree/master/Samples) were now migrated to use programmatic config. For example see [HelloWorld](https://github.com/dotnet/orleans/blob/master/Samples/HelloWorld/HelloWorld/Program.cs#L30)